### ISSUE TITLE
imgscalr.com respond 404 Not Found

### ISSUE THREAD

**Issue by Makishima** [thebuzzmedia/imgscalr#115]
Hello!
Just want see imgscalr in action but http://imgscalr.com/ is not work properly.
Please fix it.

**Comment by crico** [comment]
Yes, and maven.thebuzzmedia.com is down. What is the alternative?

**Comment by thebuzzmedia** [comment]
I took imgscalr.com down because the daily traffic was very low and I don't
have time to polish it up to really expose all the functionality in
imgscalr (the library) - it was misleading and more of a tech showcase for
an article I wrote 5 years ago.

maven.thebuzzmedia.com is down because imgscalr moved into Maven central
repo - no additional "repository" tags needed, it's in the official Maven
repo! :)

----------------
Riyad
http://thebuzzmedia.com

**Comment by daboom** [comment]
So where can we find the APIs documentation?

**Comment by iarriola** [comment]
Same as @daboom, Where can we find the API documentation?

**Comment by H-Amarado** [comment]
Not sure about the documentation but the library can be found here: http://mvnrepository.com/artifact/org.imgscalr/imgscalr-lib/4.2

**Comment by ksclarke** [comment]
Documentation can be found at: http://javadoc.io/doc/org.imgscalr/imgscalr-lib
